### PR TITLE
feat: statsperform parser

### PR DESCRIPTION
**Pull request by hbiermann95**
Added a new version of the statsperform parser.

### CODE REVIEW

File: floodlight/io/statsperform.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+    Read a position data CSV file that is given in the StatsPerform format into a
+    Tuple of six floodlight.core.xy.XY Objects comprising a Tuple of six
+    floodlight.core.xy.XY objects comprising respectively the position data for the both
+    halftimes of the game for the home team, away team, and the ball.
+
+    Parameters
+    ----------
+    filepath_dat: Union[str, Path]
+        XML File where the Position data in Statsperform format is saved.
```
Comment (draabe) [inline floodlight/io/statsperform.py:131]:
```suggestion
        XML file where the position data in StatsPerform format is saved.
```

File: floodlight/io/statsperform.py [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    links: Dict[str, Dict[int, int]] or None
+        Dictionary mapping the player jersey number to his position in the data array,
```
Comment (draabe) [inline floodlight/io/statsperform.py:133]: jID to xID?

**Review by draabe — CHANGES_REQUESTED** [review]
Hi @hbiermann95, thanks a lot for the contribution! Code looks great, there are just a few changes that I would suggest at the interface and documentation level. Make sure that function and variable names as well as docstrings are explicitly and clearly indicating what the function does and the arguments mean. It should eventually be able to use the parser (at least the public bit) withouth knowing anything about StatsPerform's data format

File: floodlight/io/statsperform.py [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    pitch = Pitch.from_template(
+        "statsperform",
+        length=pi_len,
+        width=pi_wid,
+        sport="football",
+    )
+
+    # compute framerate
+    time_values = dat_df["timeelapsed"].unique()
+    framerate = 1 / (time_values[1] - time_values[0])
```
[the remark this answers is not in the document]
Comment (hbiermann95) [reply floodlight/io/statsperform.py]: It does, I will add a fixed framerate=10 in the creation of the XY and Code objects.

File: floodlight/io/statsperform.py (hunk 2 above)
Comment (hbiermann95) [reply floodlight/io/statsperform.py:127]: Yes, I refactored the comments largely based on tracab.py

**Review by draabe — CHANGES_REQUESTED** [review]

**Review by draabe — APPROVED** [review]

**Pull request merged**